src/gui/tray/usermodel.cpp

```diff
@@ -725,6 +727,13 @@ void User::removeAccount() const
     AccountManager::instance()->save();
 }
 
+void User::slotSendReplyMessage(const QString &token, const QString &message, const QString &replyTo)
+{
+    auto *talkReply = new TalkReply(_account.data());
```

Comment:
who will delete `talkReply` ?
currently you are only deleting the internal instance of `JsonApiJob`

Reply:
I decided to go for a Q pointer, is that ok?